docs/source/support/component-statuses.rst

```diff
@@ -42,6 +42,33 @@ Runtime components
    * - Component / Feature
      - Status
      - Deprecated on
+   * - **Canton**
+     -
+     -
+   * - Canton Application and Console
+     - Stable
+     -
+     - Canton Administrative APIs for participant, and domain nodes
```

Comment:
Canton Administrative APIs for participant and domain nodes (no comma)

Reply:
![popcorn](https://media.giphy.com/media/2UvAUplPi4ESnKa3W0/giphy.gif)